installation-bundle/src/InstallTool.php

```diff
@@ -346,22 +327,22 @@ public function importTemplate(string $template, bool $preserveData = false): vo
 
             foreach ($tables as $table) {
                 if (0 === strncmp($table, 'tl_', 3)) {
-                    $this->connection->query('TRUNCATE TABLE '.$this->connection->quoteIdentifier($table));
+                    $this->connection->executeStatement('TRUNCATE TABLE '.$this->connection->quoteIdentifier($table));
                 }
             }
         }
 
         $data = file(Path::join($this->projectDir, 'templates', $template));
 
         foreach (preg_grep('/^INSERT /', $data) as $query) {
-            $this->connection->query($query);
+            $this->connection->executeStatement($query);
         }
     }
 
     public function hasAdminUser(): bool
     {
         try {
-            $statement = $this->connection->query("
+            $count = $this->connection->fetchOne("
                 SELECT
                     COUNT(*) AS count
```

Comment:
```suggestion
                    COUNT(*)
```

Reply:
Changed in efd72b05bed5fb7afde25e4688f9d878c79bf440.